src/messages/mod.rs

```diff
@@ -580,6 +581,8 @@ pub enum MessageContent {
     },
     /// Update sent to Adults and Infants by Elders
     GenesisUpdate(GenesisPfxInfo),
+    /// Send from a section to the node being relocated.
```

Comment:
better to be clustered with the above `NodeApproval` ?
also, similiar to `NodeApproval`, better state it is sent from `Section X` to the node being relocated ?

Reply:
Will fix, the comment, but I'm not clear why the clustering would make more sense. In a way this message behave like GenesisUpdate, (in that it is accumulated at the single node destination), so it seem better to cluster it with it.

Reply:
I just thought they are both being `bootstrap process` related, and comment will be similar, so suggested to cluster them.
up to you to decide.

Reply:
They are very different though, one is when you join a section, the other when you leave it.

Reply:
but within a `relocate process`, it is `you leave a section, then to join another`

Reply:
There are many ways to group or not group these, can be from the point of view of elder, or relocating/joining node. I think that the fact that the message behave/are used in a similar fashion (GenesisUpdate/Relocate) is the more relevant fact.